### PR TITLE
MAINT-36782: Fix space redactors filter when user role is added from group management

### PR DESCRIPTION
**Pull request by hakermi**
Fix space redactors filter when user role is added from group management

### CODE REVIEW

File: component/core/src/main/java/org/exoplatform/social/core/listeners/SocialMembershipListenerImpl.java [hunk 1]
```diff
@@ -118,6 +118,10 @@ public void postSave(Membership m, boolean isNew) throws Exception {
             return;
           }
           spaceService.addMember(space, userName);
+        } else if (SpaceUtils.REDACTOR.equalsIgnoreCase(m.getMembershipType())) {
+          if (spaceService.isRedactor(space, userName))
+            return;
+          spaceService.addRedactor(space, userName);
```
Comment (hbenali) [inline component/core/src/main/java/org/exoplatform/social/core/listeners/SocialMembershipListenerImpl.java]:
```suggestion
        } else if (SpaceUtils.REDACTOR.equalsIgnoreCase(m.getMembershipType())) {
          if (spaceService.isRedactor(space, userName)) {
            return;
          }    
          spaceService.addRedactor(space, userName);
```

**Review by hbenali — CHANGES_REQUESTED** [review]
Please add a unit test case.

File: component/core/src/main/java/org/exoplatform/social/core/listeners/SocialMembershipListenerImpl.java (hunk 1 above)
Comment (hakermi) [reply component/core/src/main/java/org/exoplatform/social/core/listeners/SocialMembershipListenerImpl.java]: does this make difference ?

Comment (hbenali) [reply component/core/src/main/java/org/exoplatform/social/core/listeners/SocialMembershipListenerImpl.java]: No change, but I suggest being in harmony with the above implementation.

**Review by hbenali — APPROVED** [review]

**Pull request merged**